<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Furniture</class>
 <widget class="QWidget" name="Furniture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QPushButton
{
	border: 1px solid #B8B8B8;
	border-radius: 15px;
	
	background-color: rgb(51, 71, 85);
	
	
	color: rgb(255, 255, 255);
}

QPushButton:hover
{
	color: rgb(51, 71, 85);

	background-color: rgb(51, 69, 83);
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>931</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget
{
border-image: url(:/resources/background/3_furn2Background.png);
}</string>
   </property>
   <widget class="QPushButton" name="btn_Sbed">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>主卧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_corr">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>走廊</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_book">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>书房</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>183</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">/*下拉列表框*/
QComboBox
{
    background:transparent;
    padding-left:5px ;
    border-top-left-radius:3px;
    border-top-right-radius:3px;
    border: 1px solid rgb(209 , 209 , 209);
}
QComboBox:hover
{
    border: 1px solid rgb(21 , 131 , 221);
}

QComboBox QAbstractItemView::item
{
    height:40px;
}
QComboBox::down-arrow
{
	
	
    border-image: url(:/resources/down.png) 0 34 0 0 ;
}
/*
QComboBox::down-arrow:hover
{
    border-image: url(:/resources/down.png) 0 17 0 17 ;
}
QComboBox::down-arrow:on
{
    border-image: url(:/resources/down.png) 0 0 0 34 ;
}
*/
QComboBox::drop-down
{
    width:20px;
    background:transparent; /*不加此句下拉箭头背景色为灰色与整体样式不一致，也可设置 border:0px; border-radius:0px; background:white; border-left:0px ; 即设置为无边框*/
    padding-right:5px;
}
</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string notr="true">灯光控制</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::NoInsert</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>灯光控制</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/1.png</normaloff>:/resources/1.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>窗户控制</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/2.png</normaloff>:/resources/2.png</iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_Mbed">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>更纱黑体 UI SC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>次卧</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_live">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等距更纱黑体 SC</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>客厅</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_hum">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);


border: 1px solid #B8B8B8;
border-radius: 15px;
background-color: rgb(51,69,83);
color: rgb(240, 240, 240);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_tem">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);


border: 1px solid #B8B8B8;
border-radius: 15px;
background-color: rgb(51,69,83);
color: rgb(240, 240, 240);</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_smoke">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);


border: 1px solid #B8B8B8;
border-radius: 15px;
background-color: rgb(51,69,83);
color: rgb(240, 240, 240);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/background/ico_smoke.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/background/ico_tem.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/resources/background/ico_hum.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_return">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

border:none;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_return">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

border:none;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_going">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

border:none;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_going">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>470</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
   <zorder>label_going</zorder>
   <zorder>label_return</zorder>
   <zorder>btn_Sbed</zorder>
   <zorder>btn_corr</zorder>
   <zorder>btn_book</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>btn_Mbed</zorder>
   <zorder>btn_live</zorder>
   <zorder>LCD_hum</zorder>
   <zorder>LCD_tem</zorder>
   <zorder>LCD_smoke</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>btn_return</zorder>
   <zorder>pushButton</zorder>
   <zorder>btn_going</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
